### PR TITLE
DEVELOPER-1177

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

File: _ext/jboss_developer.rb [hunk 1]
```diff
@@ -113,6 +127,19 @@ def transform site, page, content
               a['class'] = classes.uniq.join ' '
               altered = true
             end
+            # Add high-value-interaction class
+            site.high_value_interactions.each do |act|
+              puts "output_path: #{page.output_path}, from: #{act[:from]}index.html, url: #{url}, to: #{act[:to]}" if page.output_path.start_with?('/products/fuse/get-started')
```
Comment (paulrobinson) [inline _ext/jboss_developer.rb]: Did you mean to leave this in:

    '/products/fuse/get-started'

Comment (paulrobinson) [inline _ext/jboss_developer.rb]: I suppose the whole line wants removing (debug).

**Comment by paulrobinson** [comment]
Is adding the 'high-value-interaction' style class enough? Don't you also need some javascript that does the Eloqua stuff, to just those links with this style?

File: _ext/jboss_developer.rb (hunk 1 above)
Comment (pmuir) [inline _ext/jboss_developer.rb]: Yes, thanks, removed!

**Comment by pmuir** [comment]
Yes, adding the class is enough. GTM inspects the link click, and fires if that class is present.

**Comment by paulrobinson** [comment]
Looks good now. Ready for me to merge?

**Comment by pmuir** [comment]
Yes.

**Pull request merged**